modules/cloud-management-platforms-ocm.adoc

```diff
@@ -9,15 +9,15 @@ To discover cluster assets from Red{nbsp}Hat OpenShift Cluster Manager, create a
 
 .Prerequisites
 * A Red{nbsp}Hat account.
-* An link:https://console.redhat.com/openshift/token[Red{nbsp}Hat OpenShift Cluster Manager API token].
+* An link:https://console.redhat.com/iam/service-accounts[Red{nbsp}Hat service account].
```

Comment:
Thank you @stehessel for this PR. 

**issue:** ⚠️ It should be `A` instead of `An`.

**question:** ❓Do they need both `A Red Hat account.` and `An [Red Hat service account](https://console.redhat.com/iam/service-accounts).`? OR either OR only Red Hat service account?

Reply:
They need both a Red Hat account and a Red Hat service account. Well technically they only need the latter, but to create a service account you also need a Red Hat account.